pretix_passbook/passbook.py

```diff
@@ -208,7 +210,7 @@ def generate(self, order_position: OrderPosition) -> Tuple[str, str, str]:
 
         passfile.description = ugettext('Ticket for {event} ({product})').format(event=ev.name, product=ticket)
         passfile.barcode = Barcode(message=order_position.secret, format=BarcodeFormat.QR)
-        passfile.barcode.altText = order_position.secret
+        passfile.barcode.altText = order_position.secret[:10].upper()
```

Comment:
Is there a specific reason you chose upper-case here? I think that might also cause confusion since the experienced pretix user knows that order codes are always upper case and secrets are always lower case ;)

Reply:
I chose upper case for readability.

Reply:
I'd definitely be confused by that, as a pretix user.

Reply:
Also, it would be nice to indicate that this is truncated by appending a `"…"`, I think.

Reply:
I understand a pretix developer could be confused when this code is presented in upper-case. I as an end-user however do not agree. I've found that the lower-case printed `altText` is less user-friendly and frankly harder to read. Besides that, I've noticed that the pretixSCAN apps allow searching for both order codes and order_position.secret. 
Differentiating between multiple tickets within a single order should perhaps be thought of outside this PR, by for example adding the name registered to the ticket to the pass.

I guess it's up to you in the end, but personally I disagree with the confusion described earlier.